x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.test.tsx

```diff
@@ -0,0 +1,165 @@
+      expect(screen.getByTestId('alertSuppressionDurationPropertyTitle')).toBeInTheDocument();
+      expect(
+        screen.getByTestId('alertSuppressionSuppressionFieldPropertyTitle')
+      ).toBeInTheDocument();
+      expect(screen.getByTestId('alertSuppressionGroupByPropertyValue')).toHaveTextContent(
+        'field1field2'
+      );
+      expect(screen.getByTestId('alertSuppressionDurationPropertyValue')).toHaveTextContent(
+        ALERT_SUPPRESSION_PER_RULE_EXECUTION
+      );
+      expect(screen.getByTestId('alertSuppressionSuppressionFieldPropertyValue')).toHaveTextContent(
+        ALERT_SUPPRESSION_SUPPRESS_ON_MISSING_FIELDS
+      );
+    });
+
+    test('should render only AlertSuppressionTitle and SuppressAlertsDuration when rule type does not have group_by field like threshold', () => {
+      jest
+        .spyOn(useAlertSuppressionMock, 'useAlertSuppression')
+        .mockReturnValueOnce({ isSuppressionEnabled: true });
+      const rule: Partial<RuleResponse> = {
+        alert_suppression: {
+          duration: { value: 2, unit: 'm' },
+          missing_fields_strategy: 'doNotSuppress' as AlertSuppressionMissingFieldsStrategy,
+        },
+      };
+
+      render(<RuleDefinitionSection rule={rule} />);
+
+      expect(screen.queryByTestId('alertSuppressionGroupByPropertyTitle')).not.toBeInTheDocument();
+      expect(
+        screen.getByTestId('alertSuppressionSuppressionFieldPropertyTitle')
+      ).toBeInTheDocument();
+      expect(screen.getByTestId('alertSuppressionDurationPropertyValue')).toHaveTextContent('2m');
+      expect(screen.getByTestId('alertSuppressionSuppressionFieldPropertyValue')).toHaveTextContent(
+        ALERT_SUPPRESSION_DO_NOT_SUPPRESS_ON_MISSING_FIELDS
+      );
+    });
+
+    test('should not render anything when isSuppressionEnabled is false', () => {
+      const rule: Partial<RuleResponse> = {
+        alert_suppression: {
+          group_by: ['field1', 'field2'],
+          duration: { value: 2, unit: 'h' },
+          missing_fields_strategy: 'doNotSuppress' as AlertSuppressionMissingFieldsStrategy,
+        },
+      };
+
+      jest
+        .spyOn(useAlertSuppressionMock, 'useAlertSuppression')
+        .mockReturnValueOnce({ isSuppressionEnabled: false });
+
+      render(<RuleDefinitionSection rule={rule} />);
+
+      expect(screen.queryByTestId('alertSuppressionGroupByPropertyTitle')).not.toBeInTheDocument();
+      expect(screen.queryByTestId('alertSuppressionDurationPropertyTitle')).not.toBeInTheDocument();
+      expect(
+        screen.queryByTestId('alertSuppressionSuppressionFieldPropertyTitle')
+      ).not.toBeInTheDocument();
+    });
+    test('should not render anything when alert_suppression property is not present in the rule', () => {
+      const rule: Partial<RuleResponse> = {};
+
+      jest
+        .spyOn(useAlertSuppressionMock, 'useAlertSuppression')
+        .mockReturnValueOnce({ isSuppressionEnabled: false });
```

Comment:
has to have `true` here, as test title implies it won't be rendered because `alert_suppression` props missing.
But previous test already asserts no suppression fields will be rendered when `isSuppressionEnabled=false`

Reply:
11077bde861